recipes/mdsplus/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "mdsplus" %}
+{% set version = "7.7.9" %}
+{% set sha256 = "5b54e4f7bb00afee7ca9ad0fd7f5fda69afe39c817f68c7739939b1415f05008" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/MDSplus/{{ name }}/archive/stable_release-{{ version | replace('.','-') }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - configure.patch.osx  # [osx]
+    - configure.patch.linux  # [linux]
+    - mdsdcl_show_version.c.patch
+    - Makefile.in.patch
+    - mdsdcl_Makefile.in.patch
+
+build:
+  number: 0
+  skip: True  # [win or py3k]
+
+requirements:
+  build:
+    - libxml2 2.9.5
+    - openmotif 2.3.7
+    - openmotif-dev 2.3.7
```

Comment:
Am hoping it does not need to be pinned that tightly. Following up to get [details about its ABI compatibility]( https://github.com/lvc/upstream-tracker/issues/25 ).